### PR TITLE
[luci-interpreter] Introduce Splitted kernel builders

### PR DESCRIPTION
**Pull request by binarman**
This PR adds splitted kernel builders file.

Related issue: #7288
Related DRAFT: #7605

Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

**Comment by m-bronnikov** [comment]
How about union header file for all kernel builders? =)

File: compiler/luci-interpreter/src/loader/KernelBuilder.cpp [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 #include "loader/KernelBuilder.h"
```
Comment (m-bronnikov) [inline compiler/luci-interpreter/src/loader/KernelBuilder.cpp:18]: As I understand, we apply definitions of builders now, but usage(building) of introduced in files implementations you will apply in next PR. If so, LGTM!

Comment (m-bronnikov) [reply compiler/luci-interpreter/src/loader/KernelBuilder.cpp:18]: (optional) Why we make this empty line? I think `KernelBuilder` and `Builders` inclusions could be together =)

**Review by m-bronnikov — APPROVED** [review]

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Comment by m-bronnikov** [comment]
Are you planning to merge this soon? =)

**Comment by binarman** [comment]
> Are you planning to merge this soon? =)

Yes =p

**Pull request merged**